### DISCUSSION TITLE
Enables User Account Control Typo?

### DISCUSSION THREAD

**Discussion by XerXes777** [memstechtips/UnattendedWinstall#106] [General]
In the end of the unattended.xml is maybe a typo?
`reg.exe add "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System" /v EnableLUA /t REG_DWORD /d 3 /f`

But only 1 and 0 are correct values. [Info](https://learn.microsoft.com/de-de/windows-hardware/customize/desktop/unattend/microsoft-windows-lua-settings-enablelua)

Is thes the correct command?
`reg.exe add "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System" /v EnableLUA /t REG_DWORD /d 1 /f`

**Comment by memstechtips** [comment]
Hi, the command is correct. It sets it to level 3, which is the default.
